includes/class-post-type-offer.php

```diff
@@ -0,0 +1,1289 @@
+<?php
```

Comment:
Post types classes can be moved to `/includes/post-types` folder for better organization.